src/vulkan/resource.h

```diff
@@ -68,6 +70,13 @@ class Resource {
            VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT;
   }
 
+  bool CheckMemoryHostCoherent(uint32_t memory_type_index) {
```

Comment:
"Check.." should be renamed to "Is...".

The "Check" is always ambiguous: i don't know what the true value means.  With "Is" it's always clear.

Reply:
done